examples/with-extracted-stylesheet/next.config.js

```diff
@@ -0,0 +1,82 @@
+const path = require('path')
+const ExtractTextPlugin = require('extract-text-webpack-plugin')
+const exportPathMap = require('./routes').exportPathMap
+
+module.exports = {
+  webpack: (config, { buildId, dev }) => {
+    config.module.rules.push({
+      test: /\.((sa|sc|c)ss|jpg|png)$/,
+      use: [
+        {
+          loader: 'emit-file-loader',
+          options: {
+            name: 'dist/[path][name].[ext]'
+          }
+        }
+      ]
+    })
+
+    config.module.rules.push({
+      test: /\.(jpg|png|svg)$/,
+      use: [
+        {
+          loader: 'url-loader',
+          options: {
+            limit: 10000,
+            name: '.static/assets/[hash].[ext]',
+            outputPath: dev ? path.join(__dirname, '/') : undefined,
+            publicPath: function (url) {
+              return url.replace(/^.*.static/, '/static')
+            }
+          }
+        }
+      ]
+    })
+
+    config.module.rules.push({
```

Comment:
guess this can be changed to @zeit/next-css now that it uses extracted-loader 🕵️